.github/workflows/main.yml

```diff
@@ -0,0 +1,88 @@
+name: Lint, Test Code and Push Docker image for DEVELOP or TAG
+
+on:
+  pull_request:
+    paths:
+      - '**'
+
+  push:
+    paths:
+      - '**'
+
+env:
+  IMAGE_NAME: opendatacube/dashboard
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
They recommend [actions/checkout@v2](https://github.com/actions/checkout) now

Reply:
I'll update this.